advanced_functionality/huggingface_byo_scripts_and_data/huggingface-custom-text-summarizer.ipynb

```diff
@@ -0,0 +1,2886 @@
+{
```

Comment:
<p>Any kernel recommendation for Studio?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3095/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>